indexer/options.go

```diff
@@ -4,15 +4,16 @@ import (
 	"net/http"
 )
 
-// Opts are options to instantiate a indexer
-type Opts struct {
+// Options are options to instantiate a indexer
```

Comment:
should we consider doing this in another PR to reduce the blast readius?

Reply:
I think it's fine to fix while we're in here.